nekoyume/Assets/_Scripts/L10n/LanguageTypeInspectorDisplayDrawer.cs

```diff
@@ -0,0 +1,9 @@
+﻿using UniRx;
+
+namespace Nekoyume.L10n
+{
+    [UnityEditor.CustomPropertyDrawer(typeof(LanguageTypeReactiveProperty))]
+    public class LanguageTypeInspectorDisplayDrawer : InspectorDisplayDrawer
+    {
+    }
+}
```

Comment:
The empty line is required here.

```suggestion
}

```